thrimshim/thrimshim/main.py

```diff
@@ -114,11 +111,14 @@ def update_row(ident, new_row):
 		return 'Row {} not found'.format(ident), 404
 	assert old_row.id == ident
 	
-	if old_row.state not in ['UNEDITED', 'EDITED', 'CLAIMED']:
+
+	if old_row.state not in ['UNEDITED', 'EDITED', 'CLAIMED'] and not ('video_link' in new_row and new_row['video_link']):
```

Comment:
video link check can be written more concisely as `new_row.get('video_link')`, which will return None (and thus be false) if the key isn't present